src/app/shared/shared.module.ts

```diff
@@ -40,34 +33,38 @@ import { ImageModule, SpinnerModule } from 'ngx-prx-styleguide';
     WysiwygComponent
   ],
   exports: [
-    AdvancedConfirmDirective,
     AdvancedSectionComponent,
     CommonModule,
     FormsModule,
-    ButtonComponent,
-    CapitalizePipe,
     CopyInputDirective,
     DatepickerComponent,
+    DatepickerModule,
     DurationPipe,
-    FancyDurationComponent,
-    FancyFieldComponent,
+    FancyFormModule,
     FileSelectDirective,
     FileSizePipe,
     FocusDirective,
+    HeroModule,
+    ImageModule,
     ImageUploadComponent,
     TabComponent,
     TimeAgoPipe,
     TimepickerComponent,
     TextOverflowEllipsesComponent,
     TextOverflowFadeComponent,
     TextOverflowWebkitLineClampComponent,
+    SpinnerModule,
     WysiwygComponent
   ],
   imports: [
     CommonModule,
     FormsModule,
     RouterModule,
+    DatepickerModule,
```

Comment:
I don't think the order should really make a difference for NgModule. I think what we're doing here is dealing with Angular stuff first then our stuff just for organization, but we didn't really follow that with exports. Probably just lack of setting and following standards.